_install-and-configure/install-dashboards/debian.md

```diff
@@ -129,5 +129,5 @@ By default, OpenSearch Dashboards, like OpenSearch, binds to `localhost` when yo
     sudo systemctl restart opensearch-dashboards
     ```
 1. From a web browser, navigate to OpenSearch Dashboards. The default port is 5601.
-1. Log in with the default username `admin` and the default password `admin`.
+1. Log in with the default username `admin` and the default password `admin`. (For OpenSearch 2.12 and later, the password should be the custom admin password)
```

Comment:
How would user's set the custom password in OpenSearch 2.12?

Reply:
I have described this [here](https://github.com/opensearch-project/documentation-website/pull/6058/files/e479f77d5c55c28278071d5218bcd89e8c87f23d#diff-cf0aa8940e6ce028434b4b8a5bafe5399d8e2200dc1b5f8abc3ed3922759c588R48) as part of OpenSearch setup, since this is OSD setup.

Reply:
Let's add a link to that section then. I'll do that during the editorial review.